pkg/auto/release.go

```diff
@@ -211,3 +208,17 @@ func (r *Release) PullIcon(ctx context.Context, rootFs billy.Filesystem) error {
 	// git reset return
 	return r.git.ResetHEAD()
 }
+
+func loadChartYaml(rootFs billy.Filesystem, chart string, chartVersion string) (*chart.Metadata, error) {
+	// Get Chart.yaml path and load it
+	chartYamlPath := path.RepositoryChartsDir + "/" + chart + "/" + chartVersion + "/Chart.yaml"
+	absChartPath := filesystem.GetAbsPath(rootFs, chartYamlPath)
+
+	// Load Chart.yaml file
+	chartMetadata, err := helmChartutil.LoadChartfile(absChartPath)
+	if err != nil {
+		return nil, fmt.Errorf("could not load %s: %s", chartYamlPath, err)
```

Comment:
I would suggest to change the `%s` to `%v` for the `err` value.
```suggestion
		return nil, fmt.Errorf("could not load %s: %v", chartYamlPath, err)
```

Reply:
Implemented in a comment what you asked, thank you for the heads-up